src/guide/1/getting-started/first-step/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: full.html.hbs
+algolia: true
+title: Kuzzle
```

Comment:
Shouldn't the title, which is shown in the left navigation panel, be "Getting Started" instead?